wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,89 @@
+
+[Project Loved](/wiki/Community/Project_Loved) organizes its workflows in different servers, divided by the ruleset which they specialize in. **The official development of the project is done in the osu! Discord server.** These hubs aim to facilitate the discussion of the mode's picks to be added. Non-game mode dependant discussions should always be carried out in `#osu-loved`, in the **osu!** Discord server.
+
+| Name | Owner |
+| :-- | :-- |
+| [osu! Project Loved](https://discord.gg/gn58Uk5sTE) | [Librarian](https://osu.ppy.sh/users/10083084) |
+| [osu!catch Project Loved](https://discord.gg/phgtyS4UCh) | [Wesley](https://osu.ppy.sh/users/2407265) |
+| [osu!mania Loved Community](https://discord.gg/Ededv7m) | [Paturages](https://osu.ppy.sh/users/1375479) |
+| [Project Loved: Taiko](https://discord.gg/GhfjtZ6) | [Hivie](https://osu.ppy.sh/users/14102976) |
+
+## Gameplay
+
+These servers are focused on giving users a place to discuss osu!'s main attraction — the gameplay! They serve as hubs for players to interact with their daily gaming ventures in osu!.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [osu! University](https://discord.gg/QubdHdnBVg ) | [DigitalHypno](https://osu.ppy.sh/users/4384207) | **osu! University** is a community Discord server with a focus on discussion about improvement, mainly in the [osu!](/wiki/Game_mode/osu!) game mode. It regularly hosts various events such as top player interviews, [tournaments](/wiki/Tournaments), and improvement-related studies, and is also well known for hosting paid coaching services. |
+| [osu! Medal Hunters](https://discord.gg/osumedalhunters) | [MegaMix](https://osu.ppy.sh/users/18152711) | **osu! Medal Hunters** is the main Discord server used by the medal hunting community, where people discuss [medals](/wiki/Medals) and [medal solutions](/wiki/Medals/Unlock_requirements). |
+
+## Development
+
+These servers focus on providing and developing tools and systems for the game. While the official development of osu! is done in the official community Discord server, certain projects require their own workspaces before proposing and merging their work in game.
+
+**The following servers are not officially affiliated with the game.** To follow up with official updates, join the [osu! Discord server](https://discord.gg/ppy) instead.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [Performance Points](https://discord.gg/aqPCnXu) | - | - |
+| [osu!catch dev server](https://discord.gg/YEJBENvFzN) | - | - |
+| [o!m SR/PP Rework Hub](https://discord.gg/GFCNNg8bwk) | [Quenlla](https://osu.ppy.sh/users/4725379) | The **osu!mania SR/PP Rework Hub** aims to offer a place to discuss active developments from the community to improve the current difficulty systems used by osu!mania. Users can view and discuss the different ideas developed by their peers before they get officially proposed. |
```

Comment:
this description seems identical to what the others would be, but just scoped to a game mode, so probably write it in a paragraph before the table?

Reply:
Wanted to outreach the rest beforehand, but sure it can be done (Makes things simpler as well)